From 1369b9c03aa169d593bd1a3033abb2c4783ad9d9 Mon Sep 17 00:00:00 2001
From: Codrin Ciubotariu <codrin.ciubotariu@microchip.com>
Date: Thu, 3 Oct 2019 18:32:18 +0300
Subject: [PATCH] launch.sh: Use start-stop-daemon to restart egt

Use start-stop-daemon to properly restart egt with parameters.

Suggested-by: Eugen Hristev <eugen.hristev@microchip.com>
Signed-off-by: Codrin Ciubotariu <codrin.ciubotariu@microchip.com>
---
 launch.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/launch.sh b/launch.sh
index 0e6b5eb..a709edb 100755
--- a/launch.sh
+++ b/launch.sh
@@ -29,7 +29,7 @@ handle_exit_key()
 run()
 {
     handle_exit_key $@
-    egt-launcher
+    /etc/init.d/egtdemo start
 }
 
 # close stdout, stderr, stdin and double fork - it's magic
-- 
2.17.1

